pipeline.conf

```diff
@@ -4,34 +4,40 @@ input {
     }
 }
 filter {
-    # For now, we have reactomews and everything else is assumed to come from reactome.org
-    if [source] =~ ".*reactomews.*" {
-        mutate {
-            add_field => {"reactome_server" => "reactomews"}
-        }
-    }
-    else {
-        mutate {
-            add_field => {"reactome_server" => "reactome.org"}
-        }
-    }
     # Fingerprint to avoid duplicate messages
     fingerprint {
         source => "message"
         target => "[@metadata][fingerprint]"
         method => "MURMUR3"
     }
-    # Parse Apache log message.
-    if [reactome_server] == "reactome.org" {
+    # Rename the "agent" field from beats beacuse it might clash with previosuly existing "agent" objects
```

Comment:
Typos - 'because' and 'previously'.  Justin and I found really good configuration options for spell checking in IntelliJ.  Eclipse probably has similar settings to help for checking comments.